dashboard/Dockerfile

```diff
@@ -5,9 +5,9 @@ COPY package.json yarn.lock /app/
 RUN yarn install --frozen-lockfile
 
 RUN mkdir /app/src
-RUN curl -L https://unpkg.com/@bitnami/hex@3.2.0/dist/hex.min.css > /app/src/hex.min.css
-
 COPY . /app
+RUN yarn run prettier-check
+RUN curl -L https://unpkg.com/@bitnami/hex@3.2.0/dist/hex.min.css > /app/src/hex.min.css
```

Comment:
why did you move the `curl`? (seems like I had a reason to do it before the copy but tbh I don't remember why)

Reply:
Because the minified css in `hex.min.css` then fails the prettier-check :)

Reply:
ah, I see, `css` files are ignored by our .gitignore, we should only checck `scss` files

Reply:
Right - updated, thanks.